python/cudf/cudf/core/column/string.py

```diff
@@ -782,6 +782,75 @@ def contains(
             )
         return self._return_or_inplace(result_col)
 
+    def like(self, pat: str, esc: str = None) -> SeriesOrIndex:
+        """
```

Comment:
Could you add this entry here in python docs too: https://github.com/rapidsai/cudf/blob/branch-22.10/docs/cudf/source/api_docs/string_handling.rst ?